tracker/tasks.py

```diff
@@ -23,6 +23,7 @@ def add_mailchimp_subscriber(email):
         not r.ok
         and r.json()['title'] != 'Member Exists'
```

Comment:
This is not a huge deal, but it looks like [`Response.json()` re-parses each time it's called](https://github.com/psf/requests/blob/bfb93d4b7d269a8735f1b216093e7e9a9fdc4517/requests/models.py#L898). I'd suggest decoding it once outside the conditional, rather than typically invoking it 3 times.